vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.h

```diff
@@ -58,7 +58,10 @@ struct WindowsModalHostComponentView
 
  private:
   HWND m_hwnd{nullptr};
-  winrt::Microsoft::ReactNative::ReactContext m_context;
+  winrt::Microsoft::ReactNative::CompositionHwndHost m_compositionHwndHost;
+  winrt::Microsoft::ReactNative::ReactContext m_reactContext;
+  winrt::Microsoft::ReactNative::Composition::Experimental::ICompositionContext m_compositionContext;
+  winrt::Windows::UI::Composition::ContainerVisual rootVisual{nullptr};
```

Comment:
Same here, will need to figure out how to handle both Windows/Microsoft namespace